modules/cnf-du-annotating-workloads.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-annotating-workloads_{context}"]
+
+= Annotating workloads
+
+In order for a workload to be isolated to partitioned resources two annotations must be present. The containing namespace must be annotated with `workload.openshift.io/allowed` and the pod must be annotated with `target.workload.openshift.io`.
+
+Required OCP platform pods and namespaces  are already annotated. Supported day 2 operators need the namespace annotated.  If you want to add extra pods to the partition, such as any user-added catalog sources, both the namespace and pod must be annotated on creation.
+
+The namespace annotation allows cluster administrators to control which workloads have access to the partitioned resources. Only pods in namespaces containing the annotation `workload.openshift.io/allowed: {<workload-type> [, <workload-type]}` will be considered. In this release, `management` is the only valid <workload-type>. Pods that are annotated and are contained in a Namespace that is not annotated will fail to admit to the cluster. In this case, an error is raised indicating the pod is forbidden because the namespace does not allow the `workload type <workload-type>`.
```

Comment:
The sentence "In this release, `management` is the only valid <workload-type>." is also included in the next paragraph with additional detail. Maybe remove it here?

Reply:
Comment accepted.